crypto/CMakeLists.txt

```diff
@@ -26,12 +26,37 @@ if(CONFIG_CRYPTO)
 
   if(CONFIG_CRYPTO_CRYPTODEV)
     list(APPEND SRCS cryptodev.c)
-  endif()
 
-  # Software AES library
+    if(CONFIG_CRYPTO_CRYPTODEV_SOFTWARE)
```

Comment:
let's use this patch instead:
https://github.com/apache/nuttx/pull/10211/files#diff-c2f6f9fb79082c57d43c2890110e27c6399436eccebdc5b2d07a252d55a8b873R29

Reply:
> let's use this patch instead: https://github.com/apache/nuttx/pull/10211/files#diff-c2f6f9fb79082c57d43c2890110e27c6399436eccebdc5b2d07a252d55a8b873R29

Sure, I've rolled back this file